examples/quickstart_jax/requirements.txt

```diff
@@ -1,4 +1,4 @@
-flwr~=1.4.0
-jax~=0.4.10
-numpy~=1.21.1
-scikit_learn~=1.2.2
+flwr>=1.0, <2.0
+jax==0.4.13
+jaxlib==0.4.13
```

Comment:
Is `jaxlib` now required?

Reply:
jaxlib is already included in the examples before, last change to pyproject.toml was 6 months ago: https://github.com/adap/flower/pull/1573